sdcm/nemesis.py

```diff
@@ -462,6 +464,24 @@ def disrupt_nodetool_decommission(self, add_node=True):
                 if new_node:
                     new_node.running_nemesis = None
 
+    def disrupt_nodetool_seed_decommission(self, add_node=True):
+        if len(self.cluster.seed_nodes) < 2:
+            raise UnsupportedNemesis("To running seed decommission the cluster must contains at least 2 seed nodes")
+        if not self.target_node.is_seed:
+            self.target_node = random.choice(self.cluster.seed_nodes)
+        target_seed_idx = self.cluster.seed_nodes.index(self.target_node)
+        self.log.info(f"Removing the seed node in index '{target_seed_idx}' from seed nodes list")
+        self.cluster.seed_nodes.pop(target_seed_idx)
+        self.target_node.is_seed = False
+
+        seed_address = ",".join([node.ip_address for node in self.cluster.seed_nodes])
```

Comment:
this is not enough, since after the new node is created, you want to add it to the seeds list

Reply:
do this also after the decommission finishes